src/input_parsers/OnnxParser.cpp

```diff
@@ -1205,14 +1217,15 @@ void OnnxParser::unsqueeze( onnx::NodeProto &node )
     {
         missingNodeError( axisName );
     }
-    Vector<SignedTensorIndex> axes = _constantIntTensors[axisName];
+    Vector<int64_t> axes = _constantIntTensors[axisName];
```

Comment:
This and the changes from 1225-1228 are the correct fix for #755. We can merge this in, or I can port the fix across to #756 if you'd like @wu-haoze ?

Reply:
Hi Matthew, merge this fix in here sounds good.